operators/community-windows-machine-config-operator/5.1.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,16 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: windows-machine-config-operator
+  operators.operatorframework.io.bundle.channels.v1: preview
+  operators.operatorframework.io.bundle.channel.default.v1: preview
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.builder: operator-sdk-v2.0.0+git
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
+  com.redhat.openshift.versions: =v4.10
```

Comment:
```suggestion
  com.redhat.openshift.versions: "v4.10"
```

Reply:
i believe `=4.10` is correct
thats what was put for 5.0.0

Reply:
Ah, ok.  I was just going by what was requested [here](https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/1262#discussion_r887655229).

Reply:
Yep, `=4.10` is correct per [link](https://redhat-connect.gitbook.io/certified-operator-guide/ocp-deployment/operator-metadata/bundle-directory/managing-openshift-versions).